backend/requirements.txt

```diff
@@ -35,19 +40,24 @@ itsdangerous==1.1.0
 Jinja2==2.11.2
 Mako==1.1.4
 MarkupSafe==1.1.1
+mccabe==0.7.0
 mongoengine==0.23.0
 mongomock==3.23.0
 msgpack==1.0.2
 mypy-extensions==0.4.3
 packaging==20.9
 pathspec==0.8.1
 pluggy==0.13.1
+promise==2.3
```

Comment:
Seems like it is needed for the graphene package for testing purposes. I'll leave it.

Reply:
Hmm okay, I think a lot of these others might not be required though? For example FCK uses MongoDB so I'm not sure if sqlalchemy would be required?

Reply:
Ok. I'll try removing them manually, and seeing the errors that are thrown

Reply:
They seemed to work. Do you think this is sufficient?